data/json/martialarts.json

```diff
@@ -1346,6 +1346,93 @@
       "spear_wood"
     ]
   },
+  {
+    "type": "martial_art",
+    "id": "style_bayonet_drill",
+    "name": { "str": "Bayonet Drill" },
+    "description": "Bayonet charge was a common tactic used during the early modern wars. It focuses on move one's body to keep or regain his own balance, and exploits weaknesses in his opponent's balance.",
+    "initiate": [
+      "You take a step forward and flex your knees.",
+      "%s enters a bayonet attack position."
+    ],
+    "learn_difficulty": 5,
+	"arm_block": 0,
+    "primary_skill": "stabbing",
```

Comment:
Are you going to add a MA manual for these to matter in this PR, or a second one?

Reply:
I was considering use existing `manual_stabbing` as a manual book. But I'm not decided yet.
Can I leave it blank?

Reply:
Can we just replicate the knife manual but with the new MA? I'd like to add this would be a good one for learning from NPCs.